lib/src/rules/use_late_for_private_fields_and_variables.dart

```diff
@@ -77,6 +77,17 @@ class _Visitor extends UnifyingAstVisitor<void> {
   CompilationUnitElement? currentUnit;
   _Visitor(this.rule, this.context);
 
+  @override
+  void visitClassDeclaration(ClassDeclaration node) {
+    // See: https://dart.dev/tools/diagnostic-messages#late_final_field_with_const_constructor
+    if (node.members.any((member) =>
+        member is ConstructorDeclaration && member.constKeyword != null)) {
```

Comment:
Consider re-writing as a for loop for performance reasons. Not sure whether the difference will be meaningful.

Reply:
Sure.  If there's any possible difference, I'll take it!